velox/connectors/hive/HiveConfig.h

```diff
@@ -31,8 +31,14 @@ class HiveConfig {
   static constexpr const char* kInsertExistingPartitionsBehavior =
       "insert_existing_partitions_behavior";
 
+  /// Maximum number of partitions per writer.
+  static constexpr const char* kMaxPartitionsPerWriters =
+      "max_partitions_per_writers";
+
   static InsertExistingPartitionsBehavior insertExistingPartitionsBehavior(
       const Config* config);
+
+  static uint32_t maxPartitionsPerWriters(const Config* baseConfig);
```

Comment:
Sorry, revised to `config`. Guess it might be from back and forth copying.

I just found out configs.rst already has the description for max_partitions_per_writers. I think it's because I forgot to remove it from the previous PR :(.